torch/onnx/_internal/fx/fx_exporter.py

```diff
@@ -298,22 +298,32 @@ def export_fx_to_onnx(
         fx_module: torch.fx.GraphModule,
         fx_module_args: Sequence[Any],
     ) -> torch.onnx.ExportOutput:
-        # Apply decomposition table to the input graph.
-        module = passes.Decompose(
-            fx_module,
-            self.decomposition_table,
-            enable_dynamic_axes=self.options.dynamic_shapes,
-        ).run(*fx_module_args)
-
         # ONNX does not support views and mutations.
         # Functionalize to get a semantically equivalent graph without mutations.
+        # NOTE: Functionalize must run before decomposition and aten graph lowering.
```

Comment:
I guess I don't understand the difference between decomposition and aten graph lowering. Could you point me to the lines?

Reply:
It seems to me that make_fx has nothing to do with aten_graph. Or aten_graph is part of decomposition table?

Reply:
In decomp pass https://github.com/pytorch/pytorch/blob/e9786149ab71874fad478109de173af6996f7eec/torch/onnx/_internal/fx/passes/decomp.py#L51-L56

In dynamo export
https://github.com/pytorch/pytorch/blob/e9786149ab71874fad478109de173af6996f7eec/torch/_dynamo/eval_frame.py#L829-L845

I'll update the comment. "aten graph lowering" is what make_fx does.

Reply:
hmmm sounds like aten_graph == using decomposition_table? If that's the case, maybe we can consider stop mentioning aten graph, just say decompose?

Reply:
it's aten_graph == make_fx, and make_fx has the option to decompose with provided decomp_table. make_fx transforms graph of torch ops to graph of aten ops. If decomp_table is provided, make_fx decomposes along the way.

Reply:
Thanks!